apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/cohorts/index.vue

```diff
@@ -150,7 +150,21 @@ const gqlFilter = computed(() => {
 
   // add hard coded page sepsific filters
   if ("all" !== route.params.catalogue) {
-    result["networks"] = { id: { equals: route.params.catalogue } };
+    result = {
```

Comment:
This filter (the or part at least) is repeated 4 times. Maybe make this a formatted string, if that's a thing in java script.